src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidatorTest.java

```diff
@@ -22,34 +27,115 @@ class RemoveValidatorTest {
     private static final String LOGGING_CONTEXT = "12345";
 
     private RemoveValidator removeValidator;
+    private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
     private RemoveDto removeDto;
+    private UpdateDto updateDto;
 
     @BeforeEach
     public void init() {
         removeValidator = new RemoveValidator();
-        removeDto = RemoveMock.getRemoveDto();
+        overseasEntitySubmissionDto = new OverseasEntitySubmissionDto();
+        removeDto = new RemoveDto();
+        updateDto = new UpdateDto();
+        overseasEntitySubmissionDto.setRemove(removeDto);
+        overseasEntitySubmissionDto.setUpdate(updateDto);
     }
     @Test
     void testNoValidationErrorReportedWhenIsNotProprietorOfLandIsNull() {
         removeDto.setIsNotProprietorOfLand(null);
+
```

Comment:
Blank lines added here and below to separate the test fixture set-up from the test call from the assertions.

Reply:
Just a suggestion. Take it or leave it but could use //Given, //When, //Then comments on the lines added.

Reply:
Yes, I've done that in the past and almost added those comments here too. Will add.

Reply:
Done.